contracts/script/Manage.s.sol

```diff
@@ -0,0 +1,496 @@
+        console2.log("verifierEstopOwner:", verifierEstopOwner);
+
+        // Locate contracts
+        TimelockController timelockController = TimelockController(payable(vm.envAddress("TIMELOCK_CONTROLLER")));
+        console2.log("Using TimelockController at address", address(timelockController));
+
+        RiscZeroVerifierRouter verifierRouter = RiscZeroVerifierRouter(vm.envAddress("VERIFIER_ROUTER"));
+        console2.log("Using RiscZeroVerifierRouter at address", address(verifierRouter));
+
+        // Deploy new contracts
+        RiscZeroGroth16Verifier verifier =
+            new RiscZeroGroth16Verifier(ControlID.CONTROL_ROOT, ControlID.BN254_CONTROL_ID);
+        console2.log("Deployed RiscZeroGroth16Verifier to", address(verifier));
+
+        RiscZeroVerifierEmergencyStop verifierEstop = new RiscZeroVerifierEmergencyStop(verifier, verifierEstopOwner);
+        console2.log("Deployed RiscZeroVerifierEmergencyStop to", address(verifierEstop));
+
+        // Schedule the 'addVerifier()' request
+        bytes4 selector = verifier.SELECTOR();
+        console2.log("selector:");
+        console2.logBytes4(selector);
+
+        uint256 scheduleDelay = vm.envOr("SCHEDULE_DELAY", timelockController.getMinDelay());
+        console2.log("scheduleDelay:", scheduleDelay);
+
+        bytes memory data = abi.encodeCall(verifierRouter.addVerifier, (selector, verifierEstop));
+
+        timelockController.schedule(address(verifierRouter), 0, data, 0, 0, scheduleDelay);
+
+        vm.stopBroadcast();
+    }
+}
+
+/// @notice Finish deployment of RISC Zero verifier with Emergency Stop mechanism.
+/// @dev Use the following environment variable to control the deployment:
+///     * TIMELOCK_CONTROLLER contract address of TimelockController
+///     * VERIFIER_ROUTER contract address of RiscZeroVerifierRouter
+///     * VERIFIER_ESTOP contract address of RiscZeroVerifierEmergencyStop
+///
+/// See the Foundry documentation for more information about Solidity scripts.
+/// https://book.getfoundry.sh/tutorials/solidity-scripting
+contract FinishDeployEstopVerifier is Script {
+    function run() external {
+        vm.startBroadcast();
+
+        // Locate contracts
+        TimelockController timelockController = TimelockController(payable(vm.envAddress("TIMELOCK_CONTROLLER")));
+        console2.log("Using TimelockController at address", address(timelockController));
+
+        RiscZeroVerifierRouter verifierRouter = RiscZeroVerifierRouter(vm.envAddress("VERIFIER_ROUTER"));
+        console2.log("Using RiscZeroVerifierRouter at address", address(verifierRouter));
+
+        RiscZeroVerifierEmergencyStop verifierEstop = RiscZeroVerifierEmergencyStop(vm.envAddress("VERIFIER_ESTOP"));
+        console2.log("Using RiscZeroVerifierEmergencyStop at address", address(verifierEstop));
+
+        RiscZeroGroth16Verifier verifier = RiscZeroGroth16Verifier(address(verifierEstop.verifier()));
```

Comment:
Another instance of https://github.com/risc0/risc0-ethereum/pull/140/files#r1628862984